packages/dapp/src/components/Account/UserInfo.tsx

```diff
@@ -13,7 +13,6 @@ const userQuery = gql`
       kycStatus
```

Comment:
Same here: remove `onfidoApplicantId` `onfidoCheckID` `kycStatus` since we no longer need kyc

Reply:
Removed.